packages/office-ui-fabric-react/src/components/Rating/Rating.styles.ts

```diff
@@ -38,7 +38,7 @@ export function getStyles(props: IRatingStyleProps): IRatingStyles {
   const ratingVerticalPadding = 8;
   const ratingHorizontalPadding = 2;
 
-  const ratingStarUncheckedColor = palette.neutralTertiary;
+  const ratingStarUncheckedColor = palette.neutralSecondary;
```

Comment:
Should these be refactored to use semantic slots as well?

Reply:
Ah, you're right. I will update